source/backend/discovery/src/lib/sdkResources/firstOrderHandlers.js

```diff
@@ -104,19 +104,14 @@ module.exports = {
 
                 return apiGatewayResources;
             },
-            [AWS_DYNAMODB_TABLE]: async ({awsRegion, resourceId, resourceName, accountId, configuration: {dbTable}}) => {
-                // Do I have access to the whole config object here? 
-
+            [AWS_DYNAMODB_TABLE]: async ({awsRegion, resourceId, resourceName, accountId, configuration}) => {
                 const {credentials} = accountsMap.get(accountId);
-                const dynamoDBClient = awsClient.createDynamoDBClient(credentials, awsRegion);
-                //Check input values, should be tableName...
-                const tableInfo = await dynamoDBClient.getTableInfo(resourceName);
-                if (!tableInfo.LatestStreamArn) {
+                if (!configuration.latestStreamArn) {
```

Comment:
Nit: I prefer to do explicit null checks `!= null` rather than rely on type coercion of 'falsey' values.